apps/site/tests/shared/wrapped-playwright.ts

```diff
@@ -8,6 +8,8 @@
   /\[Fast Refresh\]/, // Next.js dev server (for local test runs)
   /Download the Apollo DevTools for a better development experience/,
   /Download the React DevTools for a better development experience/,
+  // No idea why this is happening (happens on homepage for example), needs investigation, could be related with MUI sx prop
+  /Using kebab-case for css properties/,
```

Comment:
This is caused by `"-webkit-text-fill-color": theme.palette.gray[70]` in `mui-outlined-input-theme-options.ts:26`. It's not added by this PR, but you did add it. You need to change it to `WebkitTextFillColor: theme.palette.gray[70]`. It was added in https://github.com/blockprotocol/blockprotocol/pull/1243